src/utils.ts

```diff
@@ -0,0 +1,5 @@
+const caseInsensitiveCollator = new Intl.Collator(undefined, {
+  sensitivity: 'base',
+})
+
+export const caseInsensitiveCompare = (a: string, b: string): number => caseInsensitiveCollator.compare(a, b)
```

Comment:
We should re-export this in `index.ts` so it's available to consumers of the package.